### ISSUE TITLE
`prefer-destructured-store-props` reports error when using `$$slots`

### ISSUE THREAD

**Issue by dkrutsko** [sveltejs/eslint-plugin-svelte#463]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

8.39.0

### What version of `eslint-plugin-svelte` are you using?

2.27.3

### What did you do?

<details>
<summary>Configuration</summary>

```cjs
{
	// ...
	'svelte/prefer-destructured-store-props': 'error'
	// ...
}
```
</details>

```svelte
<script lang="ts">
...
</script>

{#if $$slots.default}
	<slot />
{/if}

<style lang="postcss">
...
</style>
```

### What did you expect to happen?

No error to be displayed?

### What actually happened?

An error was displayed.

```
Destructure default from $$slots for better change tracking & fewer redraws
```

### Link to **GitHub Repo** with Minimal Reproducible Example

[Minimal Example of Error](https://sveltejs.github.io/eslint-plugin-svelte/playground/#eJyrVkrOT0lVslKyKU4uyiwosYvJs9GHM2PyqpUz0xRUVIpz8kuK9VJS0xJLc0pqY/IUFGxAQgr6QEXV+plpQCGlWgD+Rhq6)

[Minimal Example of Fix](https://sveltejs.github.io/eslint-plugin-svelte/playground/#eJyrVkrOT0lVslKyKU4uyiwosYvJU7FS0KhWSElNSyzNKbGCMQwVahVsFVRUinPyS4o1rWPybPThOmLyqpUz0+Aqa2PyFBRsQAoV9IGy1fqZaUAhpVoAqMskmw==)

### Additional comments

The main problem is that the "fix" that's recommended results in the following error:

```
This statement is not reactive because all variables referenced in the reactive statement are immutable
```

Is this an actual error? Is the proposed fix the way to actually fix this? Why doesn't that fix work?

**Comment by ota-meshi** [comment]
Thank you for the report! I think it's bugs
